cli/src/main/java/org/jboss/as/cli/impl/aesh/cmd/security/model/InteractiveSecurityBuilder.java

```diff
@@ -376,6 +593,12 @@ protected KeyStore buildKeyStore(CommandContext ctx, boolean buildRequest) throw
     public void doFailureOccured(CommandContext ctx) throws Exception {
         // REMOVE WHEN WFCORE-3491 is fixed.
         if (keyStoreName != null) {
+            if(useLetsEncrypt && caAccount == null) {
+                ModelNode req = ElytronUtil.removeCertificateAuthorityAccount(ctx, certAuthorityAccountName);
+                SecurityCommand.execute(ctx, req, SecurityCommand.DEFAULT_FAILURE_CONSUMER);
```

Comment:
Be sure to call execute with noReload argument to be false, otherwise the server stays in reload state.